python/Iridium9602.py

```diff
@@ -265,7 +329,9 @@ def read_binary():
     #ser.write(mt_buffer)
     #ser.write(checksum_msb)
     #ser.write(checksum_lsb)
-    ser.write("%s%s%s%s%s" % (chr(len_msb), chr(len_lsb), mt_buffer,chr(checksum_msb),chr(checksum_lsb)) )
+    #ser.write("\n%s%s%s%s%s" % (chr(len_msb), chr(len_lsb), mt_buffer,chr(checksum_msb),chr(checksum_lsb)) )
```

Comment:
Delete this commented out code, and can also delete the code which calculates these variables since they aren't use